### DISCUSSION TITLE
Tab indentation in editor

### DISCUSSION THREAD

**Discussion by benniledl** [modxcms/revolution#16262] [Ideas]
Hello,
I love ModX, however when adding a new line of code in the editor I hate that I cannot indent it using the tab key.
I would really love if code formatting could be enhanced somehow :)

**Comment by smg6511** [comment]
You simply need to incorporate an editor Extra into your installation of MODX, such as Ace or Codemirror.

**Reply by benniledl** [reply]
thx I now installed Ace even tho I don't really like installing extras because I dont want some extra to produce errors but maybe im just too paranoid xd
